us-geological-survey.csl

```diff
@@ -121,8 +150,8 @@
               <et-al term="and others"/>
               <label form="short" prefix=" "/>
             </names>
-            <text variable="container-title" prefix=" " suffix=","/>
-            <text variable="collection-title" prefix=" " suffix=","/>
+            <text variable="event" prefix=" " suffix=" --"/>
```

Comment:
@rmzelle @adam3smith Hi guys, sorry about the confusion here - I am new to github and haven't used it very much yet so I'm woefully unfamiliar with pull requests and the like. I was also on mobile at the time the question of the en dash came up. I tried to update the pull request last week with the en dash, but it looks like I didn't do it correctly - I just updated the pull request and the Travis CI is running now.